From 41e45531e485e310d51820a8a500bcf18dc49c94 Mon Sep 17 00:00:00 2001
From: Jon Miranda <jonmiranda@google.com>
Date: Tue, 17 Jul 2018 13:49:12 -0700
Subject: [PATCH 03/10] Increase max distance for folder creation for tablets.

Tablets generally have larger cell sizes (relative to phones, and relative to
the icon sizes with the cells). By increasing the distance from 55% to 75% of
icon size, it is easier for users to meet the folder creation threshold before
the icon reorder timeout is triggered.

Bug: 110796219
Change-Id: I8264390b8510340f4062e05ec12d0755e93a80d8
---
 src/com/android/launcher3/Workspace.java | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/com/android/launcher3/Workspace.java b/src/com/android/launcher3/Workspace.java
index b570acaa3..de7e6d8cd 100644
--- a/src/com/android/launcher3/Workspace.java
+++ b/src/com/android/launcher3/Workspace.java
@@ -286,7 +286,9 @@ public class Workspace extends PagedView<WorkspacePageIndicator>
         mInsets.set(insets);
 
         DeviceProfile grid = mLauncher.getDeviceProfile();
-        mMaxDistanceForFolderCreation = (0.55f * grid.iconSizePx);
+        mMaxDistanceForFolderCreation = grid.isTablet
+                ? 0.75f * grid.iconSizePx
+                : 0.55f * grid.iconSizePx;
         mWorkspaceFadeInAdjacentScreens = grid.shouldFadeAdjacentWorkspaceScreens();
 
         Rect padding = grid.workspacePadding;
-- 
2.17.1

